examples/swash_render/src/main.rs

```diff
@@ -63,6 +63,19 @@ fn main() {
     let bold_style = StyleProperty::FontWeight(bold);
     builder.push(&bold_style, 0..4);
 
+    builder.push_inline_box(InlineBox {
+        id: 0,
+        index: 40,
+        width: 50.0,
+        height: 50.0,
+    });
+    builder.push_inline_box(InlineBox {
+        id: 0,
```

Comment:
It's a bit strange to have two items with the same id here...

I presume it's just because you render them based on the width/height instead of actually using the id

Reply:
They now have different ids